Makefile

```diff
@@ -29,7 +29,7 @@ VERSION_LABEL ?= ${VERSION}
 # The go commands and the minimum Go version that must be used to build the app.
 GO ?= go
 GOFMT ?= $(shell ${GO} env GOROOT)/bin/gofmt
-GO_VERSION_KIALI = 1.18.7
+GO_VERSION_KIALI = 1.19.5
```

Comment:
This change does not correspond to the purpose of the PR, but to #5800 . Please try to keep separate branches when working on multiple PRs at the same time.